packages/router-generator/tests/generator/nested-layouts/routeTree.expected.ts

```diff
@@ -138,78 +144,142 @@ const JestedLayoutB3LayoutC2BarRoute = JestedLayoutB3LayoutC2BarImport.update({
 declare module '@tanstack/react-router' {
```

Comment:
@schiller-manuel I've added conditions for the tests in the generator for `fullPath`, `id`, `path`. Are there any situations missing here that I should add?